cooldowns_paladin.lua

```diff
@@ -28,7 +28,7 @@ LCT_SpellData[31884] = {
 	class = "PALADIN",
 	offensive = true,
 	defensive = true,
-	duration = 25,

```

Comment:
Mh, I'm surprised. I guess the _others_ are 25 and that's why I updated this one

Reply:
Yeah all the talents that replace the baseline are 25 sec it seems.